R/utils_render_xml.R

```diff
@@ -755,7 +755,7 @@ footnote_mark_to_xml <- function(
     xml_r(
       xml_rPr(
         xml_baseline_adj(
-          v_align = if (grepl("\\^", spec)) "superscript" else "baseline"
+          v_align = if (startsWith(spec, "^")) "superscript" else "baseline"
```

Comment:
```suggestion
          v_align = if (grepl("^", spec, fixed = TRUE)) "superscript" else "baseline"
```
This appears incorrect? Aren't we checking for the ^ character here, would `grepl("^", spec, fixed = TRUE)` be more accurate?

Reply:
Sorry for the sloppy fixes 😨